rviz_common/include/rviz_common/ros_integration/ros_component_manager.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024, Open Source Robotics Foundation.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the Willow Garage, Inc. nor the names of its
+ *       contributors may be used to endorse or promote products derived from
+ *       this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef RVIZ_COMMON__ROS_INTEGRATION__ROS_COMPONENT_MANAGER_HPP_
+#define RVIZ_COMMON__ROS_INTEGRATION__ROS_COMPONENT_MANAGER_HPP_
+
+#include "rclcpp_components/component_manager.hpp"
+#include "rclcpp_components/node_instance_wrapper.hpp"
+#include "rviz_common/visibility_control.hpp"
+
+namespace rviz_common
+{
+namespace ros_integration
+{
+class RVIZ_COMMON_PUBLIC ComponentManager : public rclcpp_components::ComponentManager{
+public:
+    using rclcpp_components::ComponentManager::ComponentManager;
+
+    void
+    add_node_to_executor(
+        rclcpp_components::NodeInstanceWrapper & node)
+    {
+        auto node_id = unique_id_++;
+        node_wrappers_[node_id] = node;
+        if (auto exec = this->executor_.lock()) {
+            exec->add_node(node.get_node_base_interface(), true);
```

Comment:
Drop `true` since it is the default arg.
```suggestion
            exec->add_node(node.get_node_base_interface());
```